main.go

```diff
@@ -227,10 +330,33 @@ func connectAction(ctx *cli.Context) error {
 		durations[BrandName] = time.Since(start)
 	}
 
-	/* 4. Show footer message */
+	/* 4. Show Configuration Profile */
+	if len(errorMessages) == 0 {
+		fmt.Printf("\n" + connectedPrefix + " Successfully connected to Red Hat!\n")
+		fmt.Printf(infoPrefix + " Additional operations may be executing on the system to activate the following features:\n")
+		// Generate a new http client configured with mutual TLS certificate
+		tlsConfig, err := DefaultConfig.CreateTLSClientConfig()
+		if err != nil {
+			errorMessages[BrandName] = fmt.Errorf("Cannot configure TLS: %w", err)
+		}
+		client, err := http.NewHTTPClient(tlsConfig)
+		if client != nil && err == nil {
+			// Get the user profile
+			profile, err := getConfProfile(client)
+			if err != nil {
+				errorMessages[BrandName] = fmt.Errorf("Cannot get the user profile: %w", err)
+			} else {
+				showConfProfile(profile)
+			}
+
+		}
+		fmt.Printf("\n" + infoPrefix + " Run the following command to check them:\n\t# journalctl -u rhcd -f \n")
+	}
```

Comment:
Again, I'm not UX designer, but I feel that this PR adds unnecessary clutter to the output of `rhc connect` command. I believe that it could be simplified. Like this:

```
[root@rhel9 rhc]# ./rhc connect --username user@company.com
Connecting rhel9 to Red Hat.
This might take a few seconds.

Password: 

● Connected to Red Hat Subscription Management
● Connected to Red Hat Insights
● Activated the rhc daemon
   ● Remote Configuration
   ● Remediations
   ● Compliance

Successfully connected to Red Hat!

Manage your Red Hat connector systems: https://red.ht/connector
```

I don't believe that it is good idea to show following hint: `Run the following command ... journalctl -u rhcd -f`, because it looks like that something went wrong.

Running this command (`journalctl -u rhcd -f`) is also not helpful too much... I'm adding example from my system:

```
[root@rhel9 rhc]# journalctl -u rhcd -f
Jan 24 11:47:00 rhel9 systemd[1]: Started Red Hat connector daemon.
Jan 24 11:59:50 rhel9 systemd[1]: Stopping Red Hat connector daemon...
Jan 24 11:59:50 rhel9 systemd[1]: rhcd.service: Deactivated successfully.
Jan 24 11:59:50 rhel9 systemd[1]: Stopped Red Hat connector daemon.
Jan 24 11:59:50 rhel9 systemd[1]: rhcd.service: Consumed 1.892s CPU time.
Jan 24 12:00:50 rhel9 systemd[1]: Started Red Hat connector daemon.
^C
```

I don't see there any useful information and it is pretty confusing.

Reply:
I like your output suggestion, @jirihnidek. Taking into consideration some of Garrett's suggestions, what about:

```
● Connected to Red Hat Subscription Management
● Connected to Red Hat Insights
● Activated Remote Configuration
● Activated Remediations
● Activated Compliance
```

Starting `rhcd.service` is implied in this case, because one of the 3 "Activated" lines are present. If one of the profile options is not enabled, we could either omit it, or print "Not activating Compliance".

Reply:
What if this list grows? We will have then more lines added to the output, isn't it better to have the list instead of new lines?